ga4gh/frontend.py

```diff
@@ -15,11 +15,13 @@
 app = api.FlaskAPI(__name__)
 
 
-def configure(config):
+def configure(config, config_file):
```

Comment:
It'll be easier to give this some default values. What about `configure(config="DefaultConfig", configFile=None)`?